CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ function(add_stratum_object_libs _TGT)
         $<TARGET_OBJECTS:stratum_lib_p4runtime_o>
         $<TARGET_OBJECTS:stratum_lib_security_o>
         $<TARGET_OBJECTS:stratum_main_o>
-        $<TARGET_OBJECTS:stratum_public_o>
+        $<TARGET_OBJECTS:stratum_public_lib_o>
```

Comment:
Duplicate change as in PR #206?

Reply:
Side effect of basing this PR on PR #206.